spec/enum_chron_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "enum_chron"
+
+class DummyRecord
+  include Mongoid::Document
```

Comment:
What happens if we don't include this?

Reply:
Mongoid's initialize has behavior I have been unable to entirely grok or duplicate. Something, something dynamic field assignment. I can remove the dependency in DummyRecord with a small change to EnumChron, but then Mongoid::Documents have duplicate calls to normalize_enum_chrons.

Reply:
OK -- fine to leave as-is then -- most of the test suite depends on mongo at this point anyway